### PR TITLE
Add Teclast P20HD overlay

### PR DESCRIPTION
**Pull request by felipecrs**
Here is the built apk:

[treble-overlay-teclast-p20hd.apk.zip](https://github.com/phhusson/vendor_hardware_overlay/files/7591939/treble-overlay-teclast-p20hd.apk.zip)

Just rename it back from zip to apk.

### CODE REVIEW

File: Teclast/P20HD/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.teclast.P20HD"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.vendor.build.fingerprint"
+		android:requiredSystemPropertyValue="+Teclast/P20HD_EEA/P20HD_EEA*"
```
Comment (felipecrs) [inline Teclast/P20HD/AndroidManifest.xml:7]: I'm just now sure if I wrote this correctly. The below is the output from the getprop command:

```console
➜ adb shell getprop ro.vendor.build.fingerprint
Teclast/P20HD_EEA/P20HD_EEA:10/QP1A.190711.020/26621:user/release-keys
```

**Comment by felipecrs** [comment]
@phhusson really sorry to ping you, but is there anything I can do to help this PR get merged?

**Comment by phhusson** [comment]
Lol no worries for pinging after a year -_-'

File: Teclast/P20HD/AndroidManifest.xml (hunk 1 above)
Comment (felipecrs) [reply Teclast/P20HD/AndroidManifest.xml:7]: @phhusson have you seen the comment above? I know you merged already, but I'm just double-checking to ensure I didn't make any mistake.

Comment (phhusson) [reply Teclast/P20HD/AndroidManifest.xml:7]: I didn't understand the message, I guessed it was irrelevant
Well I still don't understand it

maybe you meant "I'm just **not** sure..."? In that case yeah the syntax is ok

Comment (felipecrs) [reply Teclast/P20HD/AndroidManifest.xml:7]: Oh, I just realized the typo now. But yes, that's what I meant.

Thank you then!

**Comment by Edgarruiz8585** [comment]
ruizariasedgarmanuel@gmail.com

**Pull request merged**